File: objc/Project4/Project4/WindowController.m
//
//  WindowController.m
//  Project4
//
//  Created by Jinwoo Kim on 12/22/20.
//

#import "WindowController.h"

@interface WindowController ()
@end

@implementation WindowController
- (void)windowDidLoad {
    [super windowDidLoad];
    self.window.titleVisibility = NSWindowTitleHidden;
}

- (void)cancelOperation:(id)sender {
    [self.window makeFirstResponder:self.contentViewController];
}
@end

File: swift/Project4/Project4/WindowController.swift
//
//  WindowController.swift
//  Project4
//
//  Created by Jinwoo Kim on 12/22/20.
//

import Cocoa

class WindowController: NSWindowController {
    @IBOutlet weak var addressEntry: NSTextField!
    
    override func windowDidLoad() {
        super.windowDidLoad()
        window?.titleVisibility = .hidden
    }

    override func cancelOperation(_ sender: Any?) {
        window?.makeFirstResponder(self.contentViewController)
    }
}
